kano_settings/system/boards/rpi_1.py

```diff
@@ -0,0 +1,68 @@
+#
+# rpi_1.py
+#
+# Copyright (C) 2016 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Board specific settings for Raspberry Pi 1 (A/A+/B/B+)
+# TODO: Should this support the Compute Module and RPi Zero also or should
+#       another module be created which just links back to these settings?
+#
+
+DEFAULT_CONFIG = {
+    'Keyboard-continent-index': 1,
+    'Keyboard-country-index': 21,
+    'Keyboard-variant-index': 0,
+    'Keyboard-continent-human': 'america',
+    'Keyboard-country-human': 'United States',
+    'Keyboard-variant-human': 'Generic',
+    'Audio': 'Analogue',
+    'Wifi': '',
+    'Wifi-connection-attempted': False,
+    'Overclocking': 'High',
+    'Mouse': 'Normal',
+    'Font': 'Normal',
+    'Wallpaper': 'kanux-background',
+    'Parental-level': 0,
+    'Locale': 'en_US',
+    'LED-Speaker-anim': False,
+    'Use_GLX': False
+}
```

Comment:
Do we need all of these things in the board config? Eg, the wallpaper, font etc don't seem to be related to the board.